# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tvtime/linux-header.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- tvtime-1.0.2/src/videodev2.h.vanilla	2007-05-18 15:03:38.000000000 +0000
+++ tvtime-1.0.2/src/videodev2.h	2007-05-18 15:03:41.000000000 +0000
@@ -16,7 +16,6 @@
 #ifdef __KERNEL__
 #include <linux/time.h> /* need struct timeval */
 #endif
-#include <linux/compiler.h> /* need __user */
 
 /* for kernel versions 2.4.26 and below: */
 #ifndef __user
